pom.xml

```diff
@@ -215,4 +232,38 @@
         </plugins>
     </build>
 
+    <profiles>
+        <profile>
+            <id>release</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>de.jutzig</groupId>
+                        <artifactId>github-release-plugin</artifactId>
+                        <version>1.4.0</version>
+                        <configuration>
+                            <description>Description of your release</description>
+                            <releaseName>1.0 Final</releaseName>
+                            <tag>${project.version}</tag>
```

Comment:
this will create tag from the version installed in pom.xml?

Reply:
Whoa, this appeared here by accident, I eventually decided not to use this plugin.

Reply:
Tag is should be created manually with `git tag`